uint8 id                  # 1 ~ 8
float32 revolutions         # 回転
uint16 angle              # 0 ~ 8191
int16 rotational_speed    # rpm
int16 current
uint8 motor_temperature   # ℃

